src/types.tsx

```diff
@@ -430,7 +430,11 @@ export interface ScreenProps extends ViewProps {
    */
   stackPresentation?: StackPresentationTypes;
   /**
-   * Sets the status bar animation (similar to the `StatusBar` component). Requires enabling (or deleting) `View controller-based status bar appearance` in your Info.plist file on iOS.
+   * Sets the status bar animation (similar to the `StatusBar` component).
+   * On Android, setting either `fade` or `slide` will set the transition of status bar color. On iOS, this option applies to appereance animation of the status bar.
+   * Requires enabling (or deleting) `View controller-based status bar appearance` in your Info.plist file on iOS.
+   *
+   * Defaults to `fade` on iOS and `none` on Android.
```

Comment:
The remark on this "default" value statement correctness also applies here. I did quick search I haven't found us defaulting to these values. Could you provide code places where this is set?